src/PluginClient.php

```diff
@@ -28,12 +30,20 @@ class PluginClient implements HttpClient, HttpAsyncClient
     protected $plugins;
 
     /**
+     * A list of options.
+     *
+     * @var array
+     */
+    protected $options;
+
+    /**
```

Comment:
we should list and explain the options (in the main docblock text of the constructor, not in the argument)